.changeset/light-elephants-applaud.md

```diff
@@ -0,0 +1,10 @@
+---
+"stylelint": patch
+---
+
+Changed: stricter `possible` values for:
+
+- `block-no-empty`
+- `declaration-property-max-values`
+- `declaration-property-value-no-unknown`
+- `number-max-precision`
```

Comment:
[suggest] Can you change it to one line? I believe multiple lines are not supported by Changeset.